DataFormats/EgammaCandidates/src/classes_def.xml

```diff
@@ -165,16 +203,55 @@
   <class name="reco::GsfElectron::FiducialFlags" ClassVersion="10">
    <version ClassVersion="10" checksum="2564471186"/>
   </class>
-  <class name="reco::GsfElectron::ShowerShape" ClassVersion="14">
+  <class name="reco::GsfElectron::ShowerShape" ClassVersion="16">
+   <version ClassVersion="16" checksum="2009811606"/>
+   <version ClassVersion="15" checksum="2267383514"/>
```

Comment:
sorry if I missed, but why 14->16, not 14->15?

Reply:
No real reason other than many different versions of ioread and class layout were tested, and incremental increase wasn't in my mind then.

If necessary I can fix, but would first like to have a reasonably final agreement on the ioread not working on AOD issue first.